app/assets/javascripts/application.js

```diff
@@ -110,6 +111,8 @@ function onContentLoad(){
 
   var tz = jstz.determine();
   $.cookie('timezone', tz.name(), { path: '/' });
+
+  var clip = new ZeroClipboard($(".fact_clip_button"))
```

Comment:
can we generalize the name? i assume we might want to copy from other fields in general (e.g. parameter) and fact_clip is not great as its in app.js